mm2src/mm2_main/src/mm2_tests.rs

```diff
@@ -7364,3 +7364,92 @@ fn test_sign_verify_message_eth() {
 
     assert!(response.is_valid);
 }
+
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
+fn test_no_login() {
+    let coins = json!([
+        {"coin":"RICK","asset":"RICK","rpcport":8923,"txversion":4,"overwintered":1,"protocol":{"type":"UTXO"}},
```

Comment:
You can use `mm2_test_helpers::for_tests::rick_conf` here, please also create `morty_conf` there.